praw/models/reddit/redditor.py

```diff
@@ -180,6 +180,28 @@ def gild(self, months=1):
             data={"months": months},
         )
 
+    def moderated(self):
+        """Return a list of the redditor's moderated subreddits.
+
+        :returns: A ``list`` of :class:`~praw.models.Subreddit` objects.
+            Return ``[]`` if the redditor has no moderated subreddits.
+
+        Usage:
+
+        .. code:: python
+
+            for subreddit in reddit.redditor('spez').moderated():
+                print(subreddit.display_name)
+                print(subreddit.title)
+
+        """
+        d = self._reddit.get(API_PATH["moderated"].format(user=self))
```

Comment:
Can we use a more descriptive variable name here please?

Reply:
Heya! I changed it to `modded_data`. I thought about using the variable name `data` like some of the other methods in the same model, but I figured that might be confusing because `"data"` is also a key in the dictionary that is needed. So hopefully this resolves the ambiguity and is more descriptive! :)